File: solutions/c/easy/hello-world-c/solution-approach1.c
// ──────────────────────────────────────────────────
// Problem     "Hello World!" in C
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    c
// Status      Accepted
// Submitted   2026-06-04, 12:50 p.m.
// ──────────────────────────────────────────────────

#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main() 
{
    
    char s[100];
    scanf("%[^\n]%*c", &s); 
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    printf("Hello, World!\n%s",s);
     
    return 0;
}

File: solutions/python3/easy/find-second-maximum-number-in-a-list/solution-approach1.py
# ──────────────────────────────────────────────────
# Problem     Find the Runner-Up Score!  
# Difficulty  Easy
# Subdomain   Basic Data Types
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-06-28, 08:25 p.m.
# ──────────────────────────────────────────────────

if __name__ == '__main__':
    n = int(input())
    arr = map(int, input().split())
    arr = list(set(arr))     
    arr.sort()                
    print(arr[-2])            
    
